syntax = "proto3";

package Remote_xcom;

service TaskRun{
    rpc HandleTask(task_invoke) returns (task_reply) {}
}

message task_invoke{
    repeated string args = 1;
    bytes annotations = 2;
    bytes xcoms = 3;
}

message task_reply{
    bytes xcoms = 1;
}
